h/api/groups/auth.py

```diff
@@ -0,0 +1,50 @@
+def _authorized_for_group(effective_principals, group_hashid):
```

Comment:
I find this a little confusing to read. Is the `if` statement supposed to test for empty string? If it's `None`, should the argument be optional? Why is the result `True` if there's no argument?